publication-commons/src/test/java/cucumber/permissions/PermissionsRole.java

```diff
@@ -0,0 +1,44 @@
+package cucumber.permissions;
+
+import static java.util.Arrays.stream;
+import com.fasterxml.jackson.annotation.JsonValue;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public enum PermissionsRole {
+    FILE_CURATOR("file curator"),
+    FILE_CURATOR_FOR_GIVEN_FILE("file curator at x"),
+    EVERYONE("everyone"),
+    CONTRIBUTOR_FOR_GIVEN_FILE("contributor at x"),
+    OTHER_CONTRIBUTORS("other contributors", "contributor"),
```

Comment:
"other contributors" contains "contributor", maybe not necessary, but probably nice to have it there

Reply:
it var because of the file_access_upload test, so I just added an "alias" here